elasticgraph-graphql/spec/integration/elastic_graph/graphql/datastore_query/filtering_spec.rb

```diff
@@ -1157,12 +1174,12 @@ def search_datastore(**options, &before_msearch)
           expect(triple_nested_not).to match_array(ids_of(widget1, widget3))
         end
 
-        it "is ignored when set to nil" do
+        it "returns the standard always false filter when set to nil" do
```

Comment:
```suggestion
        it "matches no documents when set to `nil`" do
```

/nit the "always false filter" is more of an implementation detail where as "matches no documents" is the user observable behavior.

Reply:
Changed